lumen/tests/ai/test_export.py

```diff
@@ -0,0 +1,226 @@
+import asyncio
+import datetime as dt
+import json
+import os
+
+import pytest
+
+from panel.chat import ChatMessage
+
+from lumen.pipeline import Pipeline
+from lumen.sources.intake import IntakeSource
+from lumen.views import Table
+
+loop = asyncio.new_event_loop()
```

Comment:
Not sure this is quite right, I think there's ways to configure pytest-asyncio to create the loop.